src/components/new-request/search.vue

```diff
@@ -394,14 +419,6 @@ export default class NewSearch extends Mixins(CommonMixin) {
     return this.$conversionTypes.find(conversion => conversion.value === this.getConversionType)?.text
   }
 
-  get inputCompClass () {
-    let errorTypes = ['entity_type_cd', 'request_action_cd', 'location']
-    if (errorTypes.some(type => this.getErrors.includes(type))) {
-      return 'mt-n5'
-    }
-    return 'mt-n2'
-  }
```

Comment:
This was used with the old version of Vuetify, but it no longer needed. This is now replaced by new line 164, which works in all cases.